releases/FPWD.html

```diff
@@ -1,1623 +1,20 @@
-<!DOCTYPE html><html lang="en-US"><head>
-    <meta charset="UTF-8">
+<!DOCTYPE html>
+<html xmlns='http://www.w3.org/1999/xhtml' lang='en' xml:lang="en">
+  <head>
+    <meta charset='utf-8' />
+    <meta http-equiv="refresh" content=
+    "0; url=http://www.w3.org/TR/2015/WD-presentation-api-20150217/" />
+    <link rel="canonical" href=
+    "http://www.w3.org/TR/2015/WD-presentation-api-20150217/" />
```

Comment:
You shouldn't make dated specs canonical, btw; Only the Eds./WDs Otherwise, people will find the outdated version with google :(